src/js/components/productSupplier/create/subsections/PackageSpecification.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+import { Controller } from 'react-hook-form';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fetchQuantityUoM } from 'actions';
+import SelectField from 'components/form-elements/v2/SelectField';
+import TextInput from 'components/form-elements/v2/TextInput';
+import Subsection from 'components/Layout/v2/Subsection';
+
+const PackageSpecification = ({ control, errors }) => {
+  const dispatch = useDispatch();
+
+  const {
+    quantityUoM,
```

Comment:
nitpicky, but let's name it as `quantityUom` and change the `UoMType` to `UomType`, as imho it's more JS' way of naming things.